src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/EndpointContracts/RazorMapToDocumentEditsParams.cs

```diff
@@ -1,23 +1,24 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT license. See License.txt in the project root for license information.
 
+#nullable disable
+
 using System;
-using MediatR;
 using Microsoft.AspNetCore.Razor.LanguageServer.Protocol;
 using Microsoft.VisualStudio.LanguageServer.Protocol;
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer
 {
-    internal class RazorMapToDocumentEditsParams : IRequest<RazorMapToDocumentEditsResponse>
+    internal class RazorMapToDocumentEditsParams
     {
         public RazorLanguageKind Kind { get; set; }
```

Comment:
Can these properties by `init`-only?

Reply:
Maybe, but I'm really trying to avoid secondary changes in these PR's, so if we'd like that done I'd suggest creating a quick issue.

Reply:
Fair enough.